config/crd/kustomization.yaml

```diff
@@ -1,54 +0,0 @@
-# This kustomization.yaml is not intended to be run by itself,
-# since it depends on service name and namespace that are out of this kustomize package.
-# It should be run by config/default
-resources:
-- bases/operator.tigera.io_installations.yaml
-- bases/operator.tigera.io_tigerastatuses.yaml
-- bases/operator.tigera.io_amazoncloudintegrations.yaml
-- bases/operator.tigera.io_apiservers.yaml
-- bases/operator.tigera.io_authentications.yaml
-- bases/operator.tigera.io_compliances.yaml
-- bases/operator.tigera.io_intrusiondetections.yaml
-- bases/operator.tigera.io_logcollectors.yaml
-- bases/operator.tigera.io_logstorages.yaml
-- bases/operator.tigera.io_managementclusters.yaml
-- bases/operator.tigera.io_managementclusterconnections.yaml
-- bases/operator.tigera.io_managers.yaml
-# +kubebuilder:scaffold:crdkustomizeresource
-
-patchesStrategicMerge:
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
```

Comment:
Sounds like we could have left  the files related to webhook and certmanager because kustomize will omit them because they're commented out here? 

Not sure which approach is better...